automatic/kubescape/kubescape.nuspec

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>kubescape</id>
+    <version>2.2.5</version>
+    <packageSourceUrl>https://github.com/HollowMan6/chocolatey-packages/tree/master/automatic/kubescape</packageSourceUrl>
+    <owners>chocolatey-community,hollowman</owners>
```

Comment:
As a note on this, once the chocolatey-community user is a maintainer of the package, we normally remove the current maintainer.